lib/Froxlor/Cron/Dns/DnsBase.php

```diff
@@ -242,18 +242,15 @@ public function writeDKIMconfigs()
 					\Froxlor\FileDir::safe_exec("chmod 0644 " . escapeshellarg($pubkey_filename));
 				}
 
-
-
                 if (Settings::Get('dkim.dkim_service_type') == 'opendkim') {
                     $dkimdomains .= $domain['domain'] . "\n";
-                    $dkimsigns .= "*@" . $domain['domain'] . " " . 'dkim-' . $domain['dkim_id'] . "._domainkey." .$domain['domain'] . "\n";
-                    $dkimkeys .= 'dkim-' . $domain['dkim_id'] . "._domainkey." .$domain['domain'] . " " . $domain['domain']  . ':dkim-' . $domain['dkim_id'] . ":". $privkey_filename       ."\n";
+                    $dkimsigns .= "*@" . $domain['domain'] . " " . 'dkim' . $domain['dkim_id'] . "._domainkey." .$domain['domain'] . "\n";
```

Comment:
selector is correct here, but the filenames arent. dkim-milter wont work this way (i know, it might get dropped, but still this should be the same value everywhere to avoid confusion)